Makefile

```diff
@@ -22,37 +17,30 @@ daml_build_log = $(state_dir)/daml_build.log
 sandbox_pid := $(state_dir)/sandbox.pid
 sandbox_log := $(state_dir)/sandbox.log
 
+trigger_build := triggers/.daml/dist/da-marketplace-triggers-$(trigger_version).dar
+
 exberry_adapter_dir := exberry_adapter/bot.egg-info
 adapter_pid := $(state_dir)/adapter.pid
 adapter_log := $(state_dir)/adapter.log
 
-matching_engine_dir := matching_engine/bot.egg-info
 matching_engine_pid := $(state_dir)/matching_engine.pid
 matching_engine_log := $(state_dir)/matching_engine.log
 
-operator_bot_dir := automation/operator/bot.egg-info
 operator_pid := $(state_dir)/operator.pid
 operator_log := $(state_dir)/operator.log
 
-issuer_bot_dir := automation/issuer/bot.egg-info
-issuer_pid := $(state_dir)/issuer.pid
-issuer_log := $(state_dir)/issuer.log
-
-custodian_bot_dir := automation/custodian/bot.egg-info
 custodian_pid := $(state_dir)/custodian.pid
 custodian_log := $(state_dir)/custodian.log
 
-broker_bot_dir := automation/broker/bot.egg-info
 broker_pid := $(state_dir)/broker.pid
 broker_log := $(state_dir)/broker.log
 
-exchange_bot_dir := automation/exchange/bot.egg-info
 exchange_pid := $(state_dir)/exchange.pid
 exchange_log := $(state_dir)/exchange.log
 
 
 ### DAML server
-.PHONY: clean stop_daml_server stop_operator stop_issuer stop_custodian stop_broker stop_exchange stop_adapter stop_matching_engine
+.PHONY: clean stop_daml_server stop_operator stop_custodian stop_broker stop_exchange stop_adapter stop_matching_engine
```

Comment:
curious - why not stop issuer?

Reply:
I removed the Issuer bot entirely - it now runs in the `OperatorTrigger`, the reason being the original issuer bot acted on behalf of the Public party to find the registry contracts (which triggers cannot do). Since the `UserAdmin` is a signatory/observer to almost everything, it worked just as well to put it in the operator trigger (and saves one more step of deployment configuration in the process).